qt/scientific_interfaces/ISISReflectometry/test/RunsTable/RunsTablePresenterProcessingTest.h

```diff
@@ -175,6 +175,14 @@ class RunsTablePresenterProcessingTest : public CxxTest::TestSuite, RunsTablePre
     verifyAndClearExpectations();
   }
 
+  void testRowParentStateChangedForAllRowsInGroupComplete() {
+    auto presenter = makePresenter(m_view, oneGroupWithARowModel());
+    expectRowState(Colour::SUCCESS);
+    expectGroupState(Colour::SUCCESS);
```

Comment:
I'm a bit confused why this is passing - shouldn't the expected group colour be `CHILDREN_SUCCESS`? And actually we haven't even called `setSuccess` on the row so this should fail anyway? I'm having trouble running my tests locally so can't check but are these tests definitely being run?

Reply:
They are being run, but for some reason the EXPECT_CALL isn't working like I'd expect

Reply:
This is an existing problem. I've raised issue #33474 to cover it because it's out of scope of this PR.